app/models/moderator_action.rb

```diff
@@ -2,11 +2,13 @@
 
 class ModeratorAction < ApplicationRecord
   HIDE = "hide"
+  REPLACEUSERNAME = "replaceUsername"
```

Comment:
IMO it's best to think of these action names as short verbs that might be reused in other contexts, so here I would go with `RESET` or `RENAME`, because you could imagine needing the same action for a project or a journal post with an objectionable title.